lib/jsdom/living/nodes/HTMLFormElement-impl.js

```diff
@@ -47,14 +47,53 @@ class HTMLFormElementImpl extends HTMLElementImpl {
     super._descendantRemoved.apply(this, arguments);
   }
 
+  _getElementNodes() {
+    if (!this.hasAttributeNS(null, "id")) {
+      return domSymbolTree.treeToArray(this, {
+        filter: node => this._filterElement(node)
+      });
+    }
+
+    const id = this.getAttributeNS(null, "id");
+    return domSymbolTree.treeToArray(this.ownerDocument, {
```

Comment:
Since we want the _root_, which is not always the owner document, we should use `this.getRootNode({})`.

If you want to write a test for this, you can tell the difference using shadow DOM. However, that's pretty complicated, so I'm OK skipping that.